examples/direct-upload/generate-keys.js

```diff
@@ -0,0 +1,48 @@
+var CryptoJS = require('crypto-js')
```

Comment:
you can use node _core_ `crypto` for this: https://nodejs.org/api/crypto.html#crypto_crypto
not saying `crypto-js` is "_bad_" but I _always_ prefer to load _less_ code if I can. 😉 
by `require`-ing `crypto-js` you are loading _all_ of this: https://github.com/brix/crypto-js/tree/develop/src
Which we _really_ don't need.